README-gradle.md

```diff
@@ -0,0 +1,97 @@
+## Using iDFlakies on a Gradle project
```

Comment:
Please clarify what Gradle/Scala/Java/... versions are needed to use the Gradle plugin here

Reply:
Please move the dependency requirements to be first